<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthorsDialog</class>
 <widget class="QDialog" name="AuthorsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>147</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Author Maintenance</string>
  </property>
  <layout class="QFormLayout" name="AuthorsLayout">
   <property name="horizontalSpacing">
    <number>8</number>
   </property>
   <property name="verticalSpacing">
    <number>8</number>
   </property>
   <property name="margin">
    <number>8</number>
   </property>
   <item row="0" column="1">
    <widget class="QLineEdit" name="FirstNameEdit"/>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="DisplayEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="DisplayLabel">
     <property name="text">
      <string>Display Name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="FirstNameLabel">
     <property name="text">
      <string>First Name:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDialogButtonBox" name="AuthorButtonBox">
     <property name="toolTip">
      <string>Exit Screen</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="LastNameEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="LastNameLabel">
     <property name="text">
      <string>Last Name:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../images/openlp-2.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>AuthorButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>AuthorsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AuthorButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>AuthorsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
